src/extract/constraints.jl

```diff
@@ -91,22 +91,22 @@ function update_ConstraintFunctionsLibrary!()
     # we want: value/max_value + margin < 1
     ConstraintFunction(:required_power_electric_net, "%", dd -> abs(@ddtime(dd.balance_of_plant.power_electric_net) - dd.requirements.power_electric_net) / dd.requirements.power_electric_net, ==, 0.0, 0.2) # relative tolerance
     ConstraintFunction(:min_required_power_electric_net, "%", dd -> (@ddtime(dd.balance_of_plant.power_electric_net) - dd.requirements.power_electric_net) / dd.requirements.power_electric_net, >, 0.0)
-    ConstraintFunction(:required_flattop, "%", dd -> abs(dd.build.oh.flattop_duration - dd.requirements.flattop_duration) / dd.requirements.flattop_duration, ==, 0.0, 0.01) # relative tolerance
-    ConstraintFunction(:min_required_flattop, "%", dd -> (dd.build.oh.flattop_duration - dd.requirements.flattop_duration) / dd.requirements.flattop_duration, >, 0.0)
+    ConstraintFunction(:required_flattop_duration, "%", dd -> abs(dd.build.oh.flattop_duration - dd.requirements.flattop_duration) / dd.requirements.flattop_duration, ==, 0.0, 0.01) # relative tolerance
+    ConstraintFunction(:min_required_flattop_duration, "%", dd -> (dd.build.oh.flattop_duration - dd.requirements.flattop_duration) / dd.requirements.flattop_duration, >, 0.0)
```

Comment:
the logic for having `required` in the name is not consistent. Should we add `required` everywhere the constraint relates to `dd.requirements`, or should we remove it everywhere... since we're talking about a constraint anyways.

Reply:
Got rid of all the occurrences of "required" in the latest commit!